hera_pspec/pspecdata.py

```diff
@@ -990,11 +1022,16 @@ def pspec(self, bls1, bls2, dsets, input_data_weight='identity', norm='I',
 
         bls2 : list of baseline groups, each being a list of ant-pair tuples
 
-        dsets : length-2 tuple or list
+        dsets : length-2 tuple or length-2 list od integers or str
             Contains indices of self.dsets to use in forming power spectra, 
             where the first index is for the Left-Hand dataset and second index 
             is used for the Right-Hand dataset (see above).
 
+        pol_select : length-2 tuple or list of strings or integers
```

Comment:
`pol_select` is a slightly unusual name for a keyword argument in my opinion. What about `pols`? It's more similar to `dsets` that way.